### PR TITLE
Updated composer.lock and Makefile

### PR DESCRIPTION
**Pull request by duozersk**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,9 +18,9 @@ message = @echo "${cyan}${bold}${1}${reset}"
 
 # Define 3 users with different permissions within the container.
 # docker-www-data is applicable only for php container.
-docker-www-data = docker-compose exec --user=82:82 $(firstword ${1}) sh -c "$(filter-out $(firstword ${1}), ${1})"
-docker-wodby = docker-compose exec $(firstword ${1}) sh -c "$(filter-out $(firstword ${1}), ${1})"
-docker-root = docker-compose exec --user=0:0 $(firstword ${1}) sh -c "$(filter-out $(firstword ${1}), ${1})"
+docker-www-data = docker-compose exec --user=82:82 $(firstword ${1}) time -f"%E" sh -c "$(filter-out $(firstword ${1}), ${1})"
+docker-wodby = docker-compose exec $(firstword ${1}) time -f"%E" sh -c "$(filter-out $(firstword ${1}), ${1})"
+docker-root = docker-compose exec --user=0:0 $(firstword ${1}) time -f"%E" sh -c "$(filter-out $(firstword ${1}), ${1})"
```
Comment (duozersk) [inline Makefile:23]: This will just put the total time at the end of command execution - one line instead of 3, and less verbose.

File: Makefile [hunk 2]
```diff
@@ -46,44 +46,44 @@ clean: | up
 	$(call message,$(PROJECT_NAME): Removing vendor and web directories)
 	$(call docker-root, php rm -rf vendor)
 	$(call docker-root, php rm -rf web)
-	$(MAKE) -s down
+	@$(MAKE) -s down
```
Comment (duozersk) [inline Makefile:49]: Mutes the printing or this string into the output.

File: Makefile [hunk 3]
```diff
@@ -46,44 +46,44 @@ clean: | up
...
 
 exec:
-	# Remove the first argument from the list of make commands.
+    # Remove the first argument from the list of make commands.
```
Comment (duozersk) [inline Makefile:52]: Replaced tabs with 4 blanks - to remove it from printing into the console output and still keep it aligned in the IDE (if tab = 4 blanks)

File: composer.lock [hunk 4]
```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "bc135d31f47171de9a1a55745e62b79d",
+    "content-hash": "3aa3c5f1cf0ab18b0a901d9c0d0489bc",
```
Comment (duozersk) [inline composer.lock:7]: composer was giving a notice that the composer.lock is not up to date with composer.json - my fault; I didn't update it after moving the features and config_update modules from `require` into `require-dev`

**Review by spleshka — APPROVED** [review]

**Pull request merged**